openedx/features/redhouse_panel/api/v0/views.py

```diff
@@ -116,3 +117,26 @@ def get(self, request, pk):
             'address': site_configuration.values.get('address') if site_configuration else '',
         }
         return Response(SiteSerializer(data).data)
+
+
+class UpdateUserActiveStatus(UpdateAPIView):
+    """
+    View to change user's active status
+    Raises:
+        NotFound: Raised if user with `pk` provided in `URL` does not exist.
+    Example:
+        `PUT/PATCH: /admin-panel/api/v0/users/1/update`
+        body: {"is_active": false}
+        :returns
+            {
+                "id": "1",
+                "is_active": false
+            }
+    """
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+    authentication_classes = (SessionAuthentication, JwtAuthentication, OAuth2Authentication,)
+    permission_classes = (IsAuthenticated, IsAdminUser)
+
+    def put(self, request, *args, **kwargs):
+        return self.partial_update(request, *args, **kwargs)
```

Comment:
Does this mean I can update user's password too?

Reply:
https://github.com/REDHOUSEVE/edx-platform/pull/53